### DISCUSSION TITLE
Having trouble installing @graphql-codegen/cli

### DISCUSSION THREAD

**Discussion by aseerkt** [dotansimha/graphql-code-generator#5082] [Q&A] [answered]
I am unable to install graphql-codegen to my react project.
I tried;
`yarn add -D @graphql-codegen/cli`

But it ends with the error.
`error Received malformed response from registry for "@graphql-tools%2fgithub-loader". The registry may be down`

**Comment by ardatan** [comment, marked as answer]
It is probably the internet connection because that package is available in NPM registry as you can see;
https://www.npmjs.com/package/@graphql-tools/github-loader
